### ISSUE TITLE
Windows 10 binaries

### ISSUE THREAD

**Issue by vectorrilke** [vinaychandranvs/subgenesis#1]
Are they coming?

**Comment by vinaychandranvs** [comment]
Yep. I don't have a Windows machine to test. If you can help with testing, I can create a Binary and send it to you. @vector88

**Comment by usrme** [comment]
I can help with this; you can send me the binaries.

**Comment by sebam91** [comment]
Any news on Windows 10 binaries?
